### ISSUE TITLE
Thank you for your work. Could you please provide the specific code used for testing the GPH Benchmark parameters in the paper?

### ISSUE THREAD

**Issue by thenotshyshyshy** [BlueDyee/TF-GPH#1]

**Comment by BlueDyee** [comment]
Hi!🦘

Let me make sure I understand your question. Are you asking how to run the evaluation metrics of the GPH Benchmark (specifically, the `compute_metrics.py` script in our repository)?

If that is your question, **we are still in the process of deciding how to release our GPH Benchmark dataset. However, we could potentially release a small portion of the GPH Benchmark for demo purposes**, which could help you with the `compute_metrics.py` script.

**Comment by thenotshyshyshy** [comment]
Yes,i want to know how to use the evaluation metrics of the GPH Benchmark compute_metrics.py

**Comment by BlueDyee** [comment]
I have added some demo data and demo code. I hope that helps.🤗

**Comment by thenotshyshyshy** [comment]
thank you

**Comment by thenotshyshyshy** [comment]
Hello,i run your code,but it loading slowly is that normall
![Uploading image.png…]()
